app/code/Magento/ReleaseNotification/README.md

```diff
@@ -1,5 +1,3 @@
- # Magento_ReleaseNotification Module
```

Comment:
The issue with README was related to an extra space at the beginning of the header. There is no need to remove the entire header, only the extra space.
Change to:
`# Magento_ReleaseNotification module`

Reply:
> # Magento_ReleaseNotification module
**done**

Thanks, @dshevtsov

Reply:
@dshevtsov, you're correct about the original issue, but I think there's some inconsistency across module READMEs. Some have the title, but some don't. It makes sense to remove this because we already include the module title in the metadata for the devdocs topic where we use this content. Having duplicate titles in devdocs is not ideal.

Reply:
@jeff-matthews, there are several reasons to include the header:
1. Our template contains the header.
2. READMEs belong to the codebase at first place, so that they should have a root header there.
3. Topics on devdocs process h1 headers as titles.
4. The MRG topics are generated using custom tooling, so the automation can be customized to process READMEs differently depending on whether the h1 header is there.

Reply:
We need to update the devdocs tooling then. I don't want to see duplicate H1s in the MRG on devdocs anymore.